shared-helpers/src/auth/UseRequireLoggedInUser.ts

```diff
@@ -1,14 +1,15 @@
 import { useContext } from "react"
 import { AuthContext } from "./AuthContext"
-import { NavigationContext } from "@bloom-housing/ui-components"
+import { useRouter } from "next/router"
 
 /**
  * Require a logged in user. Waits on initial load, then initiates a redirect to `redirectPath` if user is not
  * logged in.
  */
+
 function useRequireLoggedInUser(redirectPath: string, disable?: boolean) {
   const { profile, initialStateLoaded } = useContext(AuthContext)
-  const { router } = useContext(NavigationContext)
+  const router = useRouter()
```

Comment:
With the NavigationContext router, we discovered that the `router.push` call was not working. In the NavigationContext definition, it looks like the `router.push` function is a no-op?